Util/channel/src/main/java/io/deephaven/util/channel/SeekableChannelsProviderPlugin.java

```diff
@@ -16,10 +16,18 @@ public interface SeekableChannelsProviderPlugin {
     /**
      * Check if this plugin is compatible with the given URI and config object.
      */
-    boolean isCompatible(@NotNull URI uri, @Nullable Object config);
+    default boolean isCompatible(@NotNull URI uri, @Nullable Object config) {
+        return isCompatible(uri.getScheme(), config);
+    }
+
+    boolean isCompatible(@NotNull String uriScheme, @Nullable Object config);
```

Comment:
You can add javadoc for the new methods similar to the existing ones.
Also, just continuing my earlier comments, we can remove the `isCompatible(uri, config)` methods now.